libnmstate/nm/ovs.py

```diff
@@ -219,3 +219,15 @@ def _get_slave_profiles(master_device, devices_info):
             if master and (master.get_iface() == master_device.get_iface()):
                 slave_profiles.append(active_con.props.connection)
     return slave_profiles
+
+
+def is_ovs_interface(nmdev):
```

Comment:
Using `ovs_interface` is a bit confusing. We usually use this name to refer tot he internal interface.
Maybe one of these options will make it a bit harder to get confuse:
- ovs_port_interface
- ovs_slave_interface